app/sample_manifest_excel/sample_manifest_excel/worksheet/test_worksheet.rb

```diff
@@ -17,6 +17,7 @@ class TestWorksheet < SequencescapeExcel::Worksheet::Base
 
       def initialize(attributes = {})
         super
```

Comment:
Not sure about this file. I think it should be in the specs/ or tests/ folder. Otherwise it will always be loaded even in production.